examples/kubernetes/1.14/cilium-microk8s.yaml

```diff
@@ -365,13 +361,7 @@ spec:
         # To be able to load kernel modules
       - hostPath:
           path: /lib/modules
-          type: Directory
         name: lib-modules
-        # To be able to load ip[6]tables kernel modules
```

Comment:
Sigh, outdated k8s yaml generations will bite us at some point. Maybe we should provide a tool that spits out the yamls instead. Food for thought for another discussion.